src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -283,7 +283,7 @@
 
     <Style x:Key="IconButtonTextBlockStyle"
            TargetType="TextBlock">
-        <Setter Property="Margin" Value="10,0,0,0" />
+        <Setter Property="Margin" Value="0,0,0,0" />

```

Comment:
This seems quite unrelated. Doesn't this mean that we might be using the wrong style for a contro lwhere we didn't want a margin?

Also, this style seems to be the same as "no style", because the default is "no margin"

Reply:
Sorta. Resolved locally haha. Still need to push my changes to the PR.